ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CompletionResponse.scala

```diff
@@ -85,6 +85,7 @@ object CompletionResponse {
     }
 
   private[daml] def toException(response: TrackedCompletionFailure): StatusException =
+  // TODO error codes: Adapt V2 ?
```

Comment:
Participant team

Reply:
Just like all command-tracker stuff, it is incorporated both into the ledger-api-server and the client code. As such we need to keep the runtime team in the loop.
@stefanobaghino-da FYI